### PR TITLE
[docs] Mention using multiple reporters

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @AlexanderMoskovkin

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -233,13 +233,20 @@ testcafe all tests/sample-fixture.js -r my-reporter
...
+testcafe all tests/sample-fixture.js -r json:report.json
 ```
 
+You can also use multiple reporters in a single test run. List them separated by commas.
+
+```sh
+testcafe all tests/sample-fixture.js -r spec,xunit:report.xml
+```
+
+Note that only one reporter can write to `stdout`. All other reporters must output to files.
```
Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Does the order of reporters matter? Can i use `xunit:report.xml,spec,nunit:report-nunit.xml` ?

Comment (AlexanderMoskovkin) [reply docs/articles/documentation/using-testcafe/command-line-interface.md]: yes, you can. Do you think we should highlight it there?

Comment (kirovboris) [reply docs/articles/documentation/using-testcafe/command-line-interface.md]: Yes, I mean we should add this info.

Comment (AlexanderMoskovkin) [reply docs/articles/documentation/using-testcafe/command-line-interface.md]: I'm not sure that it's significant remark but as you wish /cc @VasilyStrelyaev

Comment (VasilyStrelyaev) [reply docs/articles/documentation/using-testcafe/command-line-interface.md]: I don't think we should add it. That's intuitive enough.
We don't write that we can specify browsers or pass command line parameters in any order. Why should we say that?

Comment (kirovboris) [reply docs/articles/documentation/using-testcafe/command-line-interface.md]: Ok, then.

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**